cmake/SetupLIBXSMM.cmake

```diff
@@ -12,11 +12,13 @@ file(APPEND
   "LIBXSMM Version:  ${LIBXSMM_VERSION}\n"
   )
 
-add_library(LIBXSMM INTERFACE IMPORTED)
-set_property(TARGET LIBXSMM
+add_library(Libxsmm INTERFACE IMPORTED)
+set_property(TARGET Libxsmm
   APPEND PROPERTY INTERFACE_INCLUDE_DIRECTORIES ${LIBXSMM_INCLUDE_DIRS})
-set_property(TARGET LIBXSMM
+set_property(TARGET Libxsmm
   APPEND PROPERTY INTERFACE_LINK_LIBRARIES ${LIBXSMM_LIBRARIES})
 # LIBXSMM falls back to blas, so we need to link against Blas with it as well.
-set_property(TARGET LIBXSMM
-  APPEND PROPERTY INTERFACE_LINK_LIBRARIES ${BLAS_LIBRARIES})
+add_dependencies(
+  Libxsmm
+  Blas
```

Comment:
Does `add_dependencies` do what we want? Doesn't it only set the build order, not actually link anything? Your comment on line 20 sounds to me that we need to link Blas into any target that links Libxsmm because functions may use Blas instead. If that's correct then we probably need to
```cmake
set_property(TARGET Libxsmm
   APPEND PROPERTY INTERFACE_LINK_LIBRARIES Blas)
```